CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,80 @@
+- those we need personally, or just want to see done
+
+A surprising number of users find it hard to accept that the Renovate core team or other community contributors won't build every feature or fix every bug which they'd like to see.
+Participating in this repo is dependent on you finding a way to accept this reality, and not being pushy about it.
+
+## Maintaining Issue and Code Quality
+
+We use GitHub Discussions as an entry and triage point for Issues, and do not permit non-maintainers to directly create Issues themselves.
+If we can confirm a bug or agree with a feature idea, and we have adequate information (typically a reproduction repository) for it to be actionable, then we will convert it to an Issue.
+The purpose of the above is that Issues are mostly actionable - either by maintainers (if we prioritize them) or by community contributors.
+
+Occasionally we may decline an idea because it's too niche - we judge that the additional complexity it would add to code and/or config would burden the project more than the feature would enhance it.
+
+We apply extensive linting rules plus code review to all PRs in order to keep the code base maintainable.
+It is not enough that a feature or fix simply "works", it must also be elegant enough to be maintainable in future.
+We may apply multiple rounds of code review and suggestions, and sometimes this will feel like a burden, but ultimately our goal is to maintain a sustainable and maintainable project and this is a higher priority than whatever feature or fix you are trying to get merged.
+
+## If you have important work to get done
+
+People who work for large and/or important companies sometimes let the "pressure" of their role or task spill over into pushiness in Open Source.
+Such people sometimes find it hard to accept that maintainers cannot spend significant time helping them troubleshoot a config issue, or talk them through logs, or providing a feature or fix they need immediately.
+They are especially prone to "pinging" a maintainer with comments like `@rarkins how can we move this forward?`.
+
+Please understand that you are not our customer, and this Open Source project like any other does not owe you even the responsiveness or support you're probably already receiving.
+If you do happen to be a Mend.io customer then escalation of feature or support requests should be done via your Support or Customer Success representative and not in Open Source.
+
+## Getting more help
+
+If you need more help than this Open Source project can provide, or simply need guaranteed help, then there are two main options:
+
+1. Become a customer of Mend.io, such as purchasing Renovate Enterprise, or
+2. Engage privately with an experienced Renovate contributor on an hourly or project basis. Mend.io staff do not currently offer this, but one of Renovate's non-Mend.io employed maintainers who otherwise works voluntarily on the project is available for such consulting work.
```

Comment:
```suggestion
2. Engage privately with an experienced Renovate contributor on an hourly or project basis. Mend.io staff do not currently offer this, but one of Renovate's non-Mend.io employed maintainers ( @secustor ) who otherwise works voluntarily on the project is available for such consulting work.
```